frontend/src/pages/Dashboard.tsx

```
@@ -185,15 +136,14 @@ const Dashboard: React.FC = () => {
         console.error('❌ API Error Response:', errorText);
         throw new Error(`HTTP ${response.status}: ${response.statusText}\n${errorText}`);
       }
+      
+      const data = await response.json();
 
       const data: RepositoryResponse = await response.json();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)